src/components/Beneficiaries/index.tsx

```diff
@@ -329,7 +331,7 @@ class BeneficiariesComponent extends React.Component<Props, State> {
 
     private handleClickSelectAddress = (item: Beneficiary) => () => {
         if (item.state && item.state.toLowerCase() === 'pending') {
-            this.props.beneficiariesCreateData({ id: item.id });
+            this.props.beneficiariesCreateData({ id: item.id } as any);
```

Comment:
need to use defined type here

Reply:
Yes, but the type this func accepts is way different what we pass, we need to think how to avoid any side effects